### ISSUE TITLE
use `sep` instead of `delimiter` in DataFrame.to_csv

### ISSUE THREAD

**Issue by ritchie46** [pola-rs/polars#1415]
Make it consistent with read_csv

**Comment by ghuls** [comment]
https://github.com/pola-rs/polars/pull/1418

**Comment by ghuls** [comment]
The following functions are also using delimiter:
```
def concat_str(exprs: tp.List["pl.Expr"], delimiter: str = "") -> "pl.Expr":
def concat_list(exprs: tp.List["pl.Expr"], delimiter: str = "") -> "pl.Expr":
```

**Comment by ritchie46** [comment]
> The following functions are also using delimiter:
> 
> ```
> def concat_str(exprs: tp.List["pl.Expr"], delimiter: str = "") -> "pl.Expr":
> def concat_list(exprs: tp.List["pl.Expr"], delimiter: str = "") -> "pl.Expr":
> ```

Yeah, we could also change thos3.

**Comment by ritchie46** [comment]
I think all were fixed by @ghuls  in #1426
